src/mh-core.jl

```diff
@@ -191,6 +191,41 @@ function AbstractMCMC.step(
     return transition, transition
 end
 
+"""
+    is_symmetric_proposal(proposal::P) where P
+
+# Example:
+
+```julia
+using Distributions, AdvancedMH
+
+# Model definition.
+model = DensityModel(s -> logpdf(Normal(), s.x) + logpdf(Normal(5,.7), s.y))
+
+# Set up the proposal.
+p = (x=RandomWalkProposal(Normal(0,.5)), y=RandomWalkProposal(Normal(0,.5)))
+
+# Implementing this will skip the computation of the Hastings ratio.
+AdvancedMH.is_symmetric_proposal(proposal::typeof(p)) = true
+
+# Sample from the posterior with initial parameters.
+chain = sample(m1, MetropolisHastings(p), 100000; chain_type=Vector{NamedTuple})
+```
```

Comment:
IMO it is more important to have some explanation of what `is_symmetric_proposal` is useful for than an example - one just has to implement it for an arbitrary symmetric proposal. The example is also included in the tests, so there is no additional benefit in having a doc test here:
```suggestion
```